apps/desktop-wallet/src/features/autoUpdate/AutoUpdateSnackbar.tsx

```diff
@@ -27,12 +27,11 @@ const AutoUpdateSnackbar = () => {
   const [error, setError] = useState('')
 
   useEffect(() => {
-    if (!newVersion || requiresManualDownload) return
+    if (!newAutoUpdateVersion) return
```

Comment:
This was the 2nd bug and the reason why the progress bar was shown with 0% instead of showing the "Download" button for manual download.